lib/matplotlib/backends/qt_compat.py

```diff
@@ -47,8 +58,19 @@
     # No ETS environment or incompatible so use rcParams.
     if rcParams['backend'] == 'Qt5Agg':
```

Comment:
By this point I think we are gaurenteed that `QT_RC_MAJOR_VERSION` is defined, we should switch on that instead of having the conditional import logic again.